root/etc/cont-init.d/98-nvm

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/with-contenv bash
+
+echo "**** installing nvm ****"
+export NVM_DIR="$HOME/.nvm"
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh | bash
+
+echo "**** loading nvm ****"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+
+echo "**** installing the latest release ****"
+nvm install node
```

Comment:
Like the [documentation said](https://github.com/nvm-sh/nvm#usage) `The first version installed becomes the default. New shells will start with the default version of node`

Reply:
this is fine